docs/public/theme.css

```diff
@@ -7,62 +7,68 @@
     'Bitstream Vera Sans Mono', 'Liberation Mono', 'Nimbus Mono L', Monaco,
     'Courier New', Courier, monospace;
 
-  --color-white: #fff;
-  --color-black: #000014;
+  --color-base-white: 0, 0%;
+  --color-base-black: 240, 100%;
+  --color-base-dark-bg: 215, 28%;
+  --color-base-dark-navbar-bg: 221, 39%;
+  --color-base-light-navbar-bg: 217, 19%;
+  --color-base-gray: 215, 14%;
```

Comment:
Not sure if I love these semantic "theme" colors creeping into the color tokens. Is this because the grayscale doesn't match up to the original one?

Reply:
Yeah, the new gray palette matches pretty much with everything that is live now except the body, and navbar backgrounds. I think hsla is too good at making shades based on a specific color, so I lost that blue/gray color that us being used now. For the time being, you think I should condense the following to where its being used? I believe there are only about 3/4 different properties where we would do this. 

For example:

```css
--color-base-dark-navbar-bg: 221, 39%;
--color-dark-navbar-bg: var(--color-base-dark-navbar-bg), 11%;
```

To this: 

```css
--theme-navbar-bg: hsla(221, 39%, 11%, 1);
```

Reply:
Yeah that seems like a good solution, these can just be hardcoded in the theme. ☺️

Reply:
Maybe you can get a name for them from something like https://colornames.org/? From what I know, it should be separated into two sections of definitions: raw color names + variants of those colors -> and then those colors get semantic names for theme like `--theme-navbar-bg`